README.md

```diff
@@ -962,11 +963,11 @@ Example result:
 
 ```
 datetime                     balance
-2019-04-08 00:00:00+00:00  47.658736
-2019-04-09 00:00:00+00:00  47.658736
-2019-04-10 00:00:00+00:00  47.658736
-2019-04-11 00:00:00+00:00  47.658736
-2019-04-12 00:00:00+00:00  47.658736
+2019-04-08 00:00:00+00:00  382338.33
```

Comment:
Can you find and use a time range that has a change? For example you can take it from 2019-04-18 until 5 days later